src/main/java/cubicchunks/server/PlayerCubeMapEntry.java

```diff
@@ -147,7 +147,7 @@ public boolean sendToPlayers() {
 		if (this.sentToPlayers) {
 			return true;
 		}
-		if (this.cube == null || !this.cube.getGeneratorStage().isLastStage()) {
+		if (this.cube == null || !this.playerCubeMap.getWorld().getGeneratorPipeline().isLastStage(cube.getGeneratorStage())) {
```

Comment:
If the predefined stage LIVE is to be used in every GeneratorPipeline as the final state, this could easily be changed. However, some measure enforcing that LIVE is the final stage must be put in place.